src/ripple/protocol/STObject.h

```diff
@@ -401,7 +401,7 @@ class STObject
         if (!cf)
             throw std::runtime_error ("Wrong field type");
 
-        cf->setValue (value);
+        cf->setValue (std::move (value));
```

Comment:
@scottschurr: I think we're OK as long as V is inferred. It's when V is *not* inferred and set to a ref that we have trouble: http://melpon.org/wandbox/permlink/HRkkFFDfwxgojNke